package.json

```diff
@@ -246,9 +246,8 @@
     "@types/pify": "^5.0.4",
     "@types/qr-image": "^3.2.5",
     "@types/qs": "^6.9.18",
-    "@types/react": "^18.0.14",
-    "@types/react-datepicker": "^4.4.2",
-    "@types/react-dom": "^18.0.5",
+    "@types/react": "^19.0.0",
+    "@types/react-dom": "^19.0.0",
```

Comment:
Enforces type resolution to 19 types, else transient deps will produce a bunch of derp tsc errors despite working at runtime - note we're enforcing *types* to be 19, not react/react-dom itself